control-plane/stor-port/src/types/v0/transport/replica.rs

```diff
@@ -74,6 +74,89 @@ impl Replica {
     }
 }
 
+/// The request type to create snapshot of a replica.
+pub struct CreateReplicaSnapshot {
+    /// UUID of the replica.
+    replica: ReplicaId,
+    /// Snapshot name.
+    snap_uuid: SnapshotId,
+    /// UUID of the replica.
+    entity: uuid::Uuid,
```

Comment:
we already have replica id above, so this is not needed.

Reply:
here entity can be removed.